org.flatpak.chessmd.yml

```diff
@@ -0,0 +1,72 @@
+id: org.flatpak.chessmd
```

Comment:
re-read the documentation you acknowledged having read. This app-id is not acceptable.

And that's not the only thing wrong.